schemaless/schemaless.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2022 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package schemaless allows the creation of schema-less queries.
+// Schema-less queries are that not associated with a YANG schema,
+// it is up to the user to ensure that validity of the path and the parameterized typed.
```

Comment:
Schema-less queries are not associated with a YANG schema,
the user is free to choose any serializable type for any particular path,
although the selection must be consistent to avoid runtime type mismatch.